packages/web/src/components/molecules/BuyButton/index.tsx

```diff
@@ -20,9 +16,8 @@ const BuyButton = styled.a`
   }
 `
 
-export const BuyDevButton = ({ className }: Props) => (
```

Comment:
@Cold-A-Muse When removing the className option, we will not be able to extend the style with styled-components. Therefore, you cannot remove the className option.

Alternatively?:
```ts
export const BuyDevButton = ({ className }: Props = {}) => ...
```

Reply:
fix in df0e033790a2de3bc1d4a8ccdb582725d469f1c9